apps/browser/src/platform/browser/browser-api.ts

```diff
@@ -623,10 +625,10 @@ export class BrowserApi {
    *
    * @param value - Determines whether to enable or disable the autofill settings.
    */
-  static updateDefaultBrowserAutofillSettings(value: boolean) {
-    chrome.privacy.services.autofillAddressEnabled.set({ value });
-    chrome.privacy.services.autofillCreditCardEnabled.set({ value });
-    chrome.privacy.services.passwordSavingEnabled.set({ value });
+  static async updateDefaultBrowserAutofillSettings(value: boolean) {
+    await chrome.privacy.services.autofillAddressEnabled.set({ value });
+    await chrome.privacy.services.autofillCreditCardEnabled.set({ value });
+    await chrome.privacy.services.passwordSavingEnabled.set({ value });
```

Comment:
nbd, but I don't think these need to happen serially; could do

```suggestion
    await Promise.all([
      chrome.privacy.services.autofillAddressEnabled.set({ value }),
      chrome.privacy.services.autofillCreditCardEnabled.set({ value }),
      chrome.privacy.services.passwordSavingEnabled.set({ value }),
    ]);
```